### ISSUE TITLE
Support entity set up without using TableAttribute

### ISSUE THREAD

**Issue by tedypranolo** [neisbut/Npgsql.Bulk#3]
My context is set up using fluent configuration, so the POCOs doesn't have any TableAttributes.

**Comment by tedypranolo** [comment]
Sorry, I thought this project was supporting EF Core instead of EF6

**Comment by neisbut** [comment]
Hi @tedypranolo, actually this project does have basic EF Core support. What is the problem do you have?

**Comment by tedypranolo** [comment]
Hi @neisbut.
Npgsql.Bulk relies on TableAttributes. Specifically here
```
        private string GetTableName(Type t)
        {
            var tableAttr = t.GetCustomAttribute<TableAttribute>();
            return tableAttr.Name;
        }

        private string GetTableNameQualified(Type t)
        {
            var tableAttr = t.GetCustomAttribute<TableAttribute>();
            return NpgsqlHelper.GetQualifiedName(tableAttr.Name, tableAttr.Schema);
        }
```
Which only works if the table names are specified using table attributes and not using code first. I have cloned your project and modified it so it would work with my code first setup.
```
      public static string GetTableNameQualified(this IEntityType t)
        {
            var tableRelational = t.Relational();
            return NpgsqlHelper.GetQualifiedName(tableRelational.TableName, tableRelational.Schema);
        }
        public static string GetTableNameQualified(this DbContext context, Type t)
        {
            return context.Model.FindEntityType(t).GetTableNameQualified();
        }
```

**Comment by neisbut** [comment]
@tedypranolo  Ah, I see now. I fixed this in 0.5.13. Uploaded package to NuGet

**Comment by tedypranolo** [comment]
Thank you for the fix.
